flams/run_blast.py

```diff
@@ -48,9 +50,12 @@ def parse(title: str) -> "ModificationHeader":
 
 
 def _run_blast(input, modification, lysine_pos, lysine_range, evalue, num_threads=1):
-    # Get BLASTDB path for selected modification + get a temporary path for output
+    # Get BLASTDB name for selected modification + get a temporary path for output
     BLASTDB = flams.databases.setup.get_blastdb_path_for_modification(modification)
```

Comment:
Don't we need to add ".pdb" after database name?

Reply:
No. makeblastdb generates 9 files with various extensions, out of which one is .pdb. It accesses all of them so we do not need to specify any extension.
We only use .pdb in database setup to check if the given database exists

Reply:
ack